packages/context/src/objects/objects.ts

```diff
@@ -43,6 +43,37 @@ export type ObjectInstance = {
     [key: string]: unknown;
 };
 
+export type FormioInputTypes = {
```

Comment:
Don't think this needs to be plural, but beyond that, I think it can just be `FormioInput` or `FormIOInput`, don't really need to say it's a type, e.g. just like we don't say `ObjectInstanceType`.

Reply:
Done.